eva_cttv_pipeline/trait_mapping/ols.py

```diff
@@ -6,6 +6,13 @@
 from eva_cttv_pipeline.trait_mapping.utils import request_retry_helper
 
 
+OLS_EFO_SERVER = 'https://www.ebi.ac.uk/ols'
+# The setting for local OLS installation should be uncommented if necessary. Note that the link
+# for the local deployment is different from the production link in three regards: (1) it must use
+# HTTP instead of HTTPS; (2) it must include the port which you used when deploying the Docker
+# container; (3) it does *not* include /ols in its path.
+# OLS_EFO_SERVER = 'http://172.22.70.138:8080'
```

Comment:
Because this is a rare event that we have to set up a local OLS installation, I don't think it is worth it to make this a configurable variable through parser options; too much effort for something that will probably not happen again very often.

Reply:
If this is local, wouldn't it be best to use `http://127.0.0.1:8080`?

Reply:
For me it wasn't exactly local, since I was running the pipeline in the cluster, and the Docker image was deployed to my machine, so I had to specify the internal IP address of my machine. But you're right, it's better to leave 127.0.0.1 as a default. Fixed.